src/hera.cpp

```diff
@@ -357,9 +358,11 @@ evmc_result hera_execute(
       case EVM_FALLBACK:
         ret.status_code = EVMC_REJECTED;
         return ret;
-      default:
+      case EVM_REJECT:
         ret.status_code = EVMC_FAILURE;
         return ret;
+      default:
+        heraAssert(false, "");
```

Comment:
Maybe make the message "Invalid execution mode" or something, even if this will probably never happen